whipper/image/cue.py

```diff
@@ -137,9 +136,9 @@ def parse(self):
                 minutes = int(m.expand('\\2'))
                 seconds = int(m.expand('\\3'))
                 frames = int(m.expand('\\4'))
-                frameOffset = frames \
-                    + seconds * common.FRAMES_PER_SECOND \
-                    + minutes * common.FRAMES_PER_SECOND * 60
+                frameOffset = int(frames
```

Comment:
Why is the explicit `int()` casting needed here?

Reply:
Don't know but I think it's unneeded...